src/patternfly/components/Nav/nav.scss

```diff
@@ -717,8 +475,26 @@
   font-size: var(--pf-c-nav__section-title--FontSize);
   font-weight: var(--pf-c-nav__section-title--FontWeight);
   color: var(--pf-c-nav__section-title--Color);
+  border-bottom: var(--pf-c-nav__section-title--BorderBottomWidth) var(--pf-c-nav__section-title--BorderBottomStyle) var(--pf-c-nav__section-title--BorderBottomColor);
+}
+
+// Toggle caret
+.pf-c-nav__toggle {
+  flex: none;
+  padding-right: var(--pf-c-nav__toggle--PaddingRight);
+  padding-left: var(--pf-c-nav__toggle--PaddingLeft);
+  margin-left: auto;
+  font-size: var(--pf-c-nav__toggle--FontSize);
+  line-height: 1;
+  pointer-events: none;
+  border: 0;
```

Comment:
looks like these are legacy styles, but are `border` and `pointer-events` needed? Looks like it might have been a button at some point but isn't now as far as I can tell.